legal-api/src/legal_api/resources/v2/namerequest.py

```diff
@@ -48,12 +48,6 @@ def get(identifier):
 def validate_with_contact_info(identifier):
     """Return a JSON object with name request information."""
     try:
-        # The request must include email or phone number
-        email = request.args.get('email', None)
-        phone = request.args.get('phone', None)
-        if not (email or phone):
-            return make_response(jsonify(message='The request must include email or phone number.'), 403)
-
         nr_response = namex.query_nr_number(identifier)
         # Errors in general will just pass though,
         # 404 is overriden as it is giving namex-api specific messaging
```

Comment:
I think best practise is to not provide a "found" or "not found" answer before checking authn/authz.

@argush3 What do you think about it?

Reply:
I think this is ok because we don't really have auth checks on this endpoint directly.  i.e. a check whether you can use the endpoint at all.  If that was the case it should block it before even querying for the NR.

But in this case I think what the current code does in this PR make sense.